### PR TITLE
Created validation fields for courses

### PR DESCRIPTION
**Pull request by dmtrth25**
Created validation fields with helper text for create new course page

https://github.com/ita-social-projects/SpaceToStudy-Client/assets/56305508/79d7ff1c-5b88-4a99-84bf-b2a612f24968

**Resolved: #1493**

### CODE REVIEW

File: src/constants/translations/en/my-courses-page.json [hunk 1]
```diff
@@ -38,5 +38,10 @@
     "applyFilters": "Apply filters",
     "categoryLabel": "Category",
     "subjectLabel": "Subject"
+  },
+  "errorMessages": {
```
Comment (Tolik170) [inline src/constants/translations/en/my-courses-page.json:42]: already have these errors in offer-page.json, maybe make sense to move them for example to common.js

File: src/containers/my-courses/course-toolbar/CourseToolbar.tsx [hunk 2]
```diff
@@ -129,35 +156,37 @@ const CourseToolbar = ({
...
-              <Typography component='span' sx={styles.weightBox}>
-                {t('myCoursesPage.filterLabel.filterItems')}
-              </Typography>
-              {t('myCoursesPage.filterLabel.and')}
-              <Typography component='span' sx={styles.weightBox}>
-                {t('myCoursesPage.filterLabel.level')}
-              </Typography>
-              {t('myCoursesPage.filterLabel.courseTemplate')}
+            {t('myCoursesPage.filterLabel.determine')}
+            <Typography component='span' sx={styles.weightBox}>
```
Comment (Tolik170) [inline src/containers/my-courses/course-toolbar/CourseToolbar.tsx:182]: use ComponentEnum for span

File: src/constants/translations/en/my-courses-page.json (hunk 1 above)
Comment (dmtrth25) [reply src/constants/translations/en/my-courses-page.json]: > already have these errors in offer-page.json, maybe make sense to move them for example to common.json

moved

File: src/containers/my-courses/course-toolbar/CourseToolbar.tsx [hunk 3]
```diff
@@ -9,24 +9,32 @@ import MenuItem from '@mui/material/MenuItem'
...
 import Checkbox from '@mui/material/Checkbox'
+import FormHelperText from '@mui/material/FormHelperText'
 
 import AppTextField from '~/components/app-text-field/AppTextField'
-import { subjectService } from '~/services/subject-service'
-import { categoryService } from '~/services/category-service'
 import AppToolbar from '~/components/app-toolbar/AppToolbar'
 import AsyncAutocomplete from '~/components/async-autocomlete/AsyncAutocomplete'
 
+import { subjectService } from '~/services/subject-service'
```
Comment (Tolik170) [inline src/containers/my-courses/course-toolbar/CourseToolbar.tsx:18]: services should be with components

**Review by Tolik170 — APPROVED** [review]

**Review by abalanovsky — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**